### ISSUE TITLE
Using capabilities instead of suid

### ISSUE THREAD

**Issue by pete3215** [projectatomic/bubblewrap#184]
To further reduce the risk of using bubblewrap with suid, it should be possible to use capabilities instead of suid root to run namespaces. If privileged user namespaces are available, it should be possible to only use the capabilities needed to use user namespaces and from there all other namespaces are available in the new user namespace.

**Comment by cgwalters** [comment]
We used to do that, but we need `CAP_SYS_ADMIN` which is really "everything" anyways.  The internal bwrap code uses privilege separation.

My instinct here is that our attack surface is going to be dominated by having `CAP_SYS_ADMIN` and for that matter things like `CAP_NET_ADMIN`.  We're just not going to benefit from dropping other random ones much.

Closing, but without prejudice - if you have a specific counter-argument referencing the code and/or a particular set of capabilities, we can look in detail.
